pkg/services/ngalert/eval/eval.go

```diff
@@ -124,7 +124,8 @@ func GetExprRequest(ctx AlertExecCtx, data []models.AlertQuery, now time.Time) (
 		OrgId: ctx.OrgID,
 		Headers: map[string]string{
 			// Some data sources check this in query method as sometimes alerting needs special considerations.
-			"FromAlert": "true",
+			"FromAlert":    "true",
+			"X-Skip-Cache": "true",
```

Comment:
```suggestion
			"X-Cache-Skip": "true",
```

Reply:
thanks. it looks like the other one from yesterday is fine, but i need ☕ this morning. fixed with 0522aad